### ISSUE TITLE
NGINX container fails to get updated configuration from Artifactory upon upgrade. Version 6.8.2

### ISSUE THREAD

**Issue by opsroller** [jfrog/charts#223]
**Is this a request for help?**: Yes

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Helm and Kubernetes**:

**Which chart**: jfrog/artifactory-ha

**What happened**: When upgrading via Helm, my updated NGINX containers fail to download the NGINX configuration snippet from artifactory.

**What you expected to happen**: The configuration downloads, validates and applies if necessary.

**How to reproduce it** (as minimally and precisely as possible): Install the 6.7.2 version of the chart, and the upgrade to 6.8.2

**Anything else we need to know**:

Values used while launching.
```
initContainerImage: "alpine:3.8"
appVersion: 6.8.2

nginx:
  enabled: true
  name: nginx
  replicaCount: 3
  image:
    repository: "docker.bintray.io/jfrog/nginx-artifactory-pro"
    pullPolicy: Always
  service:
    type: LoadBalancer
  externalPortHttp: 80
  internalPortHttp: 80
  externalPortHttps: 443
  internalPortHttps: 443
  internalPortReplicator: 6061
  externalPortReplicator: 6061
  ## The following settings are to configure the frequency of the liveness and readiness probes
  livenessProbe:
    enabled: true
    initialDelaySeconds: 100
    failureThreshold: 10
    timeoutSeconds: 10
    periodSeconds: 10
    successThreshold: 1
  readinessProbe:
    enabled: true
    initialDelaySeconds: 60
    failureThreshold: 10
    timeoutSeconds: 10
    periodSeconds: 10
    successThreshold: 1
  env:
    skipAutoConfigUpdate: false
  resources:
    requests:
      memory: "4Gi"
    limits:
      memory: "16Gi"
  customConfigMap: nginx-config
  affinity:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
        - matchExpressions:
          - key: kops.k8s.io/instancegroup
            operator: In
            values:
            - nginx-us-west-2a
            - nginx-us-west-2b
            - nginx-us-west-2c
```

**Comment by eldada** [comment]
@opsroller - there is a bug in the new Nginx Docker image (released in 6.8.0). It's missing the `curl` tools, which is used to pull the Nginx config from Artifactory. 
A fix is supposed to be released in the coming days.
Until a fix is released, you can use the older Nginx image with the new Artifactory version.
Set `nginx.image.version` to `6.7.3` (last good Artifactory Nginx version). The update functionality should work as before with the new Artifactory version.

**Comment by eldada** [comment]
This is now fixed with https://github.com/jfrog/charts/pull/269
